README.md

```diff
@@ -1 +1,3 @@
-# Mirte_Frame
\ No newline at end of file
+# Mirte_Frame
+
+Please see the MIRTE docs.
```

Comment:
Maybe add a link somewhere

Reply:
There is nothing there either about the frame at the moment. So changed to TODO.